whois-commons/src/main/resources/nrtm_schema.sql

```diff
@@ -0,0 +1,58 @@
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT = @@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS = @@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION = @@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8 */;
+/*!40103 SET @OLD_TIME_ZONE = @@TIME_ZONE */;
+/*!40103 SET TIME_ZONE = '+00:00' */;
+/*!40014 SET @OLD_UNIQUE_CHECKS = @@UNIQUE_CHECKS, UNIQUE_CHECKS = 0 */;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS = @@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS = 0 */;
+/*!40101 SET @OLD_SQL_MODE = @@SQL_MODE, SQL_MODE = 'NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES = @@SQL_NOTES, SQL_NOTES = 0 */;
+
+DROP TABLE IF EXISTS `source`;
+create table `source`
+(
+    `id`   int(11) unsigned NOT NULL AUTO_INCREMENT,
+    `name` varchar(40)      NOT NULL DEFAULT '',
+    PRIMARY KEY (`id`),
+    UNIQUE KEY `source__name_uk` (`name`)
+) ENGINE = InnoDB
+  DEFAULT CHARSET = latin1;
+
+DROP TABLE IF EXISTS `version_information`;
+create table `version_information`
+(
+    `id`         int(11) unsigned NOT NULL AUTO_INCREMENT,
```

Comment:
Do you need an `id` column?

Reply:
not for the code itself but i think in general it's always a good idea to use one in the db: you never know when someone might need to make the other columns non-unique and then you'll need an ID. it's always much harder to add later, whereas it's really easy to remove. so basically the reason is 'experience says use an explicit ID' but i'm ok to remove it if it bothers someone.

Reply:
OK